geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -88,11 +89,27 @@ object AccumuloRunner extends Runner {
 
     val params = Option(command.params)
 
+    // TODO: handle OptionalCredentialsParams
```

Comment:
Well I didn't really know what it was intended to do. The Accumulo data store always requires a username (or connection) so I wasn't quite sure what this parameter flavour  could ever be used for. Will just remove the comment I think.